coderd/coderd.go

```diff
@@ -1547,6 +1550,9 @@ func New(options *Options) *API {
 			r.Use(apiKeyMiddleware)
 			r.Get("/", api.tailnetRPCConn)
 		})
+		r.Route("/init-script", func(r chi.Router) {
+			r.Get("/", api.initScript)
+		})
```

Comment:
I know I'm the one who suggested `/api/v2/init-script`, but I don't like the idea of the `os` and `arch` being optional query parameters (and defaulting to `linux/amd64`). Could we do something like `/api/v2/init-script/:os/:arch` instead? Or always require the query parameters?